docs/src/maths/propagators.md

```diff
@@ -20,7 +20,8 @@
  - The **pullback** takes wobbliness information with respect to the function's output, and tells the equivalent wobbliness with respect to the functions input.
 
 #### More formally
-The **pushforward** of ``f`` takes the _sensitivity_ of the input of ``f`` to a quantity, and gives the _sensitivity_ of the output of ``f`` to that quantity
+The **pushforward** of ``f`` takes the _sensitivity_ of the input of ``f`` to a quantity, and gives the _sensitivity_ of the output of ``f`` to that quantity.
```

Comment:
This is how it looks currently

![image](https://github.com/JuliaDiff/ChainRulesCore.jl/assets/19503980/b06adcfa-6016-4407-bfbb-73fdb0dad128)